index.js

```diff
@@ -7,7 +7,7 @@ module.exports = function recommendations (options, overrides) {
     throw new Error('No `options` object passed')
   }
 
-  var config = _.assign(defaults, {
+  var config = _.assign({}, defaults, {
```

Comment:
In general, assigning as it was done should always be a red flag. Not only does it overwrite the defaults object, which gets shared between instantiations, but it points the config variable to the default object, so all instances of recs will share config. The last instantiation of recs will set the config for all instantiations of recs.